state-chain/pallets/cf-elections/src/electoral_systems/tests/monotonic_change.rs

```diff
@@ -80,17 +76,19 @@ fn generate_votes(
 
 fn generate_votes_with_different_slots(
 	correct_voters: AuthorityCount,
-	correct_value: MonotonicChangeVote<Value, Slot>,
+	correct_value: Value,
 	incorrect_voters: AuthorityCount,
 	incorrect_value: MonotonicChangeVote<Value, Slot>,
 ) -> ConsensusVotes<SimpleMonotonicChange> {
 	ConsensusVotes {
+		// we start from 1 so don't get a Default::default() conflict on vote validity.
 		votes: (0..correct_voters)
```

Comment:
This comment is kind of confusing - says 'start from 1' but the range starts from 0?

Reply:
stale, removing

Reply:
stale, removed